### PR TITLE
feat: make new home page visible along with sidebar

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Closes https://github.com/near/near-discovery/issues/1090

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -55,7 +57,7 @@ const HomePage: NextPageWithLayout = () => {
         <NotificationsAlert />
 
         <ComponentWrapperPage
-          src={components.wrapper}
+          src={sidebarLayoutHasInitialized ? components.gateway.home : components.wrapper}
```
Comment (shelegdmitriy) [inline src/pages/index.tsx:60]: @calebjacob I'm not sure this is the best way to show this new home page for logged in users along with sidebar, so feel free to leave your thoughts about that

File: src/pages/index.tsx [hunk 2]
```diff
@@ -55,7 +57,7 @@ const HomePage: NextPageWithLayout = () => {
         <NotificationsAlert />
 
         <ComponentWrapperPage
-          src={components.wrapper}
+          src={sidebarLayoutHasInitialized ? components.gateway.homePage : components.wrapper}
```
Comment (shelegdmitriy) [inline src/pages/index.tsx:60]: @calebjacob I'm not sure this is the best way to show this new home page for logged in users along with sidebar, so feel free to leave your thoughts about that

**Comment by calebjacob** [comment]
This looks great @shelegdmitriy! I just pushed a small refactor of some of my existing code to expose a new `useSidebarLayoutEnabled()` hook to help drive the home page component.

I also made a small change so that we make sure we always wrap with `component.wrapper` to make sure things like TOS get pulled in:

```tsx
<ComponentWrapperPage
          src={components.wrapper}
          componentProps={{
            logOut,
            targetProps: router.query,
            targetComponent: sidebarLayoutEnabled ? components.gateway.homePage : components.default,
            termsDomainName,
            privacyDomainName,
            recordToC: setTosData,
          }}
        />
```

**Review by calebjacob — APPROVED** [review]

**Comment by shelegdmitriy** [comment]
Thank you @calebjacob! Looks good 🙂

**Pull request merged**